CIP-0068/README.md

```diff
@@ -0,0 +1,223 @@
+---
+CIP:
+Title: Datum Metadata Standard
+Authors: Alessandro Konrad <alessandro.konrad@live.de>, Thomas Vellekoop <thomas.vellekoop@iohk.io>
+Comments-URI:
+Status: Draft
+Type: Informational
+Created: 2022-07-13
+Post-History:
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This proposal defines a metadata standard for native assets making use of output datums not only for NFTs but any asset class.
+
+## Motivation
+
+[CIP-0025](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0025) has plenty of limitations, to name a few
+
+- zero programmability
+- it is very hard to update/evolve the metadata
+- the metadata are not readable from a plutus validator. 
+
+This standard tackles all these problems and offers many more advantages, not only for NFTs, but also for any asset class that may follow. Additionally, this CIP will introduce a way to classify tokens so that third parties like wallets can easily know what the kind of token it is.
+
+## Considerations
+
+The basic idea is to have two assets issued, where one references the other. We call these two a `reference NFT` and an `user token`, where the `user token` can be an NFT, FT or any other asset class that is transferable and represents any value. So, the `user token` is the actual asset that lives in a user's wallet.
+
+To find the metadata for the `user token` you need to look for the output, where the `reference NFT` is locked in. How this is done concretely will become clear below. Moreover, this output contains a datum, which holds the metadata. The advantage of this approach is that the issuer of the assets can decide how the transaction output with the `reference NFT` is locked and further handled. If the issuer wants complete immutable metadata, the `reference NFT` can be locked at the address of an unspendable script. Similarly, if the issuer wants the NFTs/FTs to evolve or wants a mechanism to update the metadata, the `reference NFT` can be locked at the address of a script with arbitrary logic that the issuer decides.
+
+Lastly and most importantly, with this construction, the metadata can be used by a PlutusV2 script with the use of reference inputs [CIP-0031](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0031). This will drive further innovation in the token space. 
+
+## Specification
+
+This is the registered `asset_name_label` value
+
+| asset_name_label            | class        | description                                               |
+| --------------------------- | ------------ | --------------------------------------------------------- |
+| 100                         | NFT          | Reference NFT locked at a script containing the datum     |
```

Comment:
That is true because it's part of [CIP-0067](https://github.com/cardano-foundation/CIPs/pull/298) which was proposed at the same time as this CIP. It's more of placeholder for now, but gives a better understanding of the idea.